actions/post_actions.ts

```diff
@@ -292,7 +293,7 @@ export function deleteAndRemovePost(post: Post) {
             });
         }
 
-        if (post.id === getSelectedPostCardId(getState() as GlobalState)) {
+        if (post.id === getSelectedPostCardId(state as GlobalState)) {
```

Comment:
We need to get the changed state after dispatching `SELECT_POST` so calling `getState()` again is needed in this case.

Reply:
hmm, my mistake, I didn't read the full block, it seems that the previous action won't affect the next one.

Still, I don't really get what we are trying to achieve here..